site/docs/v1/tech/lambdas/_saml-response-fields.adoc

```diff
@@ -21,11 +21,10 @@ The instant that this assertion stops being valid. This is the number of millise
 [field]#samlResponse.assertion.issuer# [type]#[String]#::
 The issuer of this SAML assertion. This defaults to the `issuser` from the corresponding SAML identity provider configuration.
 
-[field]#samlResponse.assertion.subject.nameID.format# [type]#[String]#::
-The `NameID` format for the id of the subject (user). FusionAuth uses the `emailaddress` format specified by the SAML specifications. It is not recommended that you change this unless you know what you are doing.
-
-[field]#samlResponse.assertion.subject.nameID.id# [type]#[String]#::
-The `NameID` id of the subject (user). This defaults to the user's email address. It is not recommended that you change this unless you know what you are doing.
+[field]#samlResponse.assertion.subject.nameIDs# [type]#[Array<Object>]#::
+The array of `NameID` for the subject. Each `NameID` has two fields:
+ - `format` - The `NameID` format for the id of the subject (user). FusionAuth uses the `emailaddress` format specified by the SAML specifications. It is not recommended that you change this unless you know what you are doing.
+ - `id` - The `NameID` id of the subject (user). This defaults to the user's email address. It is not recommended that you change this unless you know what you are doing.
```

Comment:
We should also have a since field.

We don't really have breaking changes in our APIs, but should try to illustrate the response as it was before this change too. That is, leave the stuff you deleted in, but mark it as only valid for `version 1.XX or lower`. We have something similar here: https://fusionauth.io/docs/v1/tech/apis/two-factor (look for `1.33.0`).

Also, I think it'd be good to note this as a breaking change whenever we updated the saml library and broke this functionality in the release notes.